src/style.js

```diff
@@ -5,6 +5,11 @@ const update = typeof window === 'undefined' ? doRender : updateOnClient
 let components = []
 let updatePromise
 
+// includes desktop and iOS, based on
+// https://github.com/DamonOehlman/detect-browser/blob/master/lib/detectBrowser.js
+const isSafari = typeof window === 'undefined' ?
+  false : /Version\/[0-9._]+.*Safari/.test(window.navigator.userAgent)
```

Comment:
I have no idea how robust is this regexp but ideally you could use [useragent](http://npmjs.com/package/useragent) which is quite battle tested or check their regexps if the package affects our lib's size.